src/variable.jl

```diff
@@ -122,12 +122,13 @@ function fix!(x::Variable)
   x.vexity = ConstVexity()
   x
 end
-function fix!(x::Variable, v)
-  # TODO: check sizes match
-  x.value = @compat Array{Float64}(size(x))
-  x.value[:] = v
+function fix!(x::Variable, v::AbstractArray)
+  size(x) == size(v) || throw(DimensionMismatch("Variable and value sizes do not match!"))
+  x.value = Array{Float64}(undef, size(x))
+  x.value[:,:] = v
```

Comment:
I don't think `[:,:]` is correct here. I think you actually want to do `x.value .= v`

Reply:
In the original code there was `x.value[:] = v` that somehow worked in 0.6 but it is definitely wrong in 1.0 so I extended it to `[:, :] as `x.value` is always a matrix and so `v` must be a matrix too based on the size checking.

Reply:
It doesn't seem to matter I guess.